temba/api/views.py

```diff
@@ -31,7 +32,17 @@ class Read(OrgObjPermsMixin, SmartReadView):
 
         def get(self, request, *args, **kwargs):
             self.object = self.get_object()
-            obj_request = [header for header in self.object.request.split('\r\n') if not header.startswith('Authorization')]
+
+            obj_request = []
+            for header in self.object.request.split('\r\n'):
+                can_add = True
+                for excluded in settings.EXCLUDED_HTTP_HEADERS:
+                    if excluded.lower() in header.lower():
+                        can_add = False
```

Comment:
you can invert the logic and dont use the new variable `can_add`. For example you can use:
```
if not excluded.lower() in header.lower():
    obj_request.append(header)
```